Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/class-names.ts

```diff
@@ -1,4 +1,4 @@
-type classNamesType = string | string[] | null | undefined;
+type classNamesType = string | string[] | null | undefined | {};
```

Comment:
Does this work?
I think the result for `{"something": "otherthing"}` will be `[object Object]`

More code should be implemented in the join. I am not sure whether this logic should be here.